src/backends/ilp-quoter/index.js

```diff
@@ -24,6 +24,7 @@ class ILPQuoter {
     this.currencyPairs = this.currencyWithLedgerPairs.map((p) => [p[0].slice(0, 3),
                                                                   p[1].slice(0, 3)])
     this.backendUri = opts.backendUri
+    this.backendStatus = 'KO'
```

Comment:
It would make sense to be consistent with whatever is chosen here also: https://github.com/interledger/five-bells-connector/pull/154/files#diff-fb64b0ab1de372089a451029c4bc9b18R6.